projects/angularx-flatpickr/src/lib/flatpickr-defaults.service.ts

```diff
@@ -111,6 +111,16 @@ export interface FlatpickrDefaultsInterface {
    */
   minDate?: string | Date;
 
+  /**
```

Comment:
Can you also add entries to the defaults here please?

https://github.com/Cryptex-Data-Solutions/angularx-flatpickr/blob/main/projects/angularx-flatpickr/src/lib/flatpickr-defaults.service.ts#L328-L336